### ISSUE TITLE
What about reload when html files changes ?

### ISSUE THREAD

**Issue by edipofederle** [bhauman/lein-figwheel#46]
This is possible on current state of lein-figwheel ?

**Comment by fterrier** [comment]
Also +1 for this.

**Comment by bhauman** [comment]
I'm revisiting this.
How about a config option called refresh-paths: that will take a list of files or directories to watch.

And then when a change occurs reload the whole application page.

**Comment by expez** [comment]
@bhauman that sounds great!  Do you use [enlive](https://github.com/cgrand/enlive) at all?  If so, would you also consider bundling these [lines of code](https://github.com/kolov/enlive-reload/blob/master/src/com/akolov/enlive_reload.clj)?

It would be nice to have a one-stop-shop for keeping stuff on disk and in the browser in sync.  Is this the path you're going down now?

**Comment by tobgu** [comment]
I was not able to get the ring.middleware.reload/wrap-reload middleware to work with figwheel (maybe it's possible? I'm very new to this and didn't manage). Would be awesome if the server side code could also be reloaded on change by figwheel. I know that I could serve that from another server but that would add another moving part.

**Comment by bhauman** [comment]
Yes you'll really want to have your own server to have that work.

You can out your simple ring handler code in the :source-paths of your cljsbuild, alongside your cljs code and then it will reload automatically. But this is a trick that only works while your sever code is simple.

Better off having a separate server.

**Comment by tobgu** [comment]
Ok, thanks for your prompt reply!

**Comment by martinskou** [comment]
ring.middleware.reload/wrap-reload seems to work fine on the server with the standard reagent-template, but it does not force a reload of the browser page.

**Comment by greywolve** [comment]
refresh-paths option would be amazing!
